src/main.ts

```
@@ -73,3 +94,8 @@
     await convertToDocusaurus(plugin).finally(() => cleanUp(plugin));
   }
 };
+
+// Test formatting
+function test() {
```

ESLint

[error] 'test' is defined but never used.

(@typescript-eslint/no-unused-vars)